chart/templates/_helpers.tpl

```diff
@@ -287,4 +287,39 @@ Get the Jaeger URL
     {{- else }}
     {{- print "jaeger-collector:4317" -}}
     {{- end }}
+{{- end -}}
+
+{{/*
+ Create a normalized etcd name based on input parameters
+ */}}
+{{- define "etcdUrl" -}}
+{{- if eq (.Values.etcd.enabled) false }}
+{{- if .Values.etcd.externalUrl }}
+{{- .Values.etcd.externalUrl }}
+{{- else }}
+{{- fail "etcd.externalUrl must be set" }}
+{{- end }}
+{{- else }}
+{{- .Release.Name }}-etcd:{{ .Values.etcd.service.port }}
+{{- end }}
+{{- end }}
+
+{{/*
+Renders init containers. If unset it sets the container image.
+*/}}
+{{- define "render_init_containers" -}}
+    {{- $containers := list }}
+    {{- $image := .Values.base.initContainers.image }}
+    {{- range .value -}}
+        {{ $container := . }}
+        {{- if not (hasKey . "image") }}
+            {{- with $image }}
+                {{- $_ := set $container "image" . }}
+            {{- else }}
+                {{- $_ := set $container "image" "docker.io/bitnami/bitnami-shell:10" }}
```

Comment:
Should the default image be configurable via values.yaml? This could pose a problem for air-gapped folks.

Reply:
hmm perhaps you misunderstood the code? the default is in the values.yaml
If not present, ex: upgrades with --reuse then we hardcode it here.

Reply:
Oh, that's fair.